tests/Functional/SalesDataProviderTest.php

```diff
@@ -54,34 +54,84 @@ protected function setUp(): void
     }
 
     /** @test */
-    public function it_provides_simple_year_sales_summary_grouped_per_month(): void
+    public function it_provides_year_sales_summary_grouped_by_year(): void
     {
-        $salesSummary = $this->getSummaryForChannel('CHANNEL');
-        $expectedMonths = $this->getExpectedMonths();
+        /** @var \DateTimeInterface $startDate */
+        $startDate = new \DateTime('2019-12-27 00:00:01');
+
+        /** @var \DateTimeInterface $startDate */
```

Comment:
```suggestion
        /** @var \DateTimeInterface $endDate */
```

Reply:
(This change has to be made several times lower in the file.)

Reply:
Why is it even needed? Any IDE should guess that `\DateTime` implements `\DateTimeInteface`.